.github/workflows/ssr-docker.yml

```diff
@@ -0,0 +1,114 @@
+name: SSR Pipeline
+
+on:
+  push:
+    branches: 
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master, develop ]
+    types: [ opened, closed ]
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  ENV: "stg"
+  ARGOCD-FILE: "values-staging.yaml"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - if: github.ref != 'refs/heads/develop'
+        uses: actions/checkout@v2
+        with:
+          ref: develop
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
+
+      - if: github.ref != 'refs/heads/develop'
+        run: git branch develop origin/develop
+      
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+
+      - name: Log in to dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+      
+      - name: Extract metadata for docker
+        if: |
+          github.event_name == 'push' ||
+          startsWith(github.ref, 'refs/tags/v') ||
+          github.event_name == 'pull_request' && github.event.action == 'opened'
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: aeternity/superhero-ui-ssr
+          tags: |
+            type=raw,value=latest,enable=${{ endsWith(GitHub.ref, 'main') }}
+            type=ref,event=tag
+            type=ref,event=pr
+      
+      - name: Build and push docker image
+        if: |
+          github.event_name == 'push' ||
+          startsWith(github.ref, 'refs/tags/v') ||
+          github.event_name == 'pull_request' && github.event.action == 'opened'
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile.ssr
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
```

Comment:
this seems to not be set, should we remove it?

Reply:
Not sure , that I understand the question correctly?

Reply:
ah nvm, its generated by the docker-meta action right?

Reply:
Yes